src/debugging.js

```diff
@@ -77,9 +77,11 @@ export function getConfig(debugging) {
 }
 config.getConfig('debugging', ({debugging}) => getConfig(debugging));
 
-export function sessionLoader(storage = window.sessionStorage) {
+export function sessionLoader() {
   let overrides;
   try {
+    if (!window.localStorage || !window.sessionStorage) throw "exception";
+    let storage = window.sessionStorage || window.localStorage;
```

Comment:
thought I had already commented here, but don't see it...
If merely accessing `window.sessionStorage` might throw an exception while `window.localStorage` is still available, then this does not solve it because the second part of the `||` would never be reached.  would need something like:
`let storage;`
`try { storage = window.sessionStorage || window.localStorage; }`
`catch (e) { storage = window.localStorage; }`